swagger_py_codegen/templates/flask/validators.tpl

```diff
@@ -85,7 +85,7 @@ def request_validate(view):
             validator = FlaskValidatorAdaptor(schema)
             result, errors = validator.validate(value)
             if errors:
-                abort(422, message='Unprocessable Entity', errors=errors)
+                abort(400, message='Invalid Request', errors=errors)
```

Comment:
Why `400` is better? Will you give some reason for it?
About the `422` status code:
http://www.bennadel.com/blog/2434-http-status-codes-for-invalid-data-400-vs-422.htm
https://developer.github.com/v3/#client-errors

Perhaps left the error handling for the user is a better choice.
We will improve it.